### DISCUSSION TITLE
Kibana dashboard

### DISCUSSION THREAD

**Discussion by marekvesely-direct** [canarytrace/documentation#71] [Synthetic] [answered]
Dobry den,

mate predpripraveny nejaky dashboard pro Kibanu? Na strankach vidim nejaky pekny dashboard, ale nikde jej nemuzu najit ke stazeni co bych nahral do Kibany.

Dekuji,

Vesely

**Comment by rdpanek** [comment, marked as answer]
Dobry den @marekvesely-direct 

vyuzijte Canarytrace Installer, ktery nastavi Elasticsearch a nahraje vizualizace a dashboardy do Kibany https://canarytrace.com/docs/features/installer/

Diky

**Reply by marekvesely-direct** [reply]
Dobry den, @rdpanek 

pustil jsem to tedy pres docker-compose, kde index prefix mame upraveny takto.
ELASTIC_INDEX_PREFIX: 'canarytrace-'
Templaty v Elasticu se nastavi spravne, ovsem pro Kibanu se tento prefix nepouzije a zustane defaultni 'c.'. Jak toto spravne prenastavit, co tomu podhodit za promennou?

Dekuji,

Vesely

**Reply by rdpanek** [reply]
`ELASTIC_INDEX_PREFIX` prefix je dobre menit v pripade, kdy vedle sebe bezi vice ruznych verzi (pro indexy elasticu). Pokud prefix nutne nepotrebujete menit (vsechny nase nastroje zacinaji timto prefixem, vcetne listeneru, spotu a automat. analyz) tak jej doporucuji nechat s defaulltni hodnotou.

objekty pro kibanu maji tento prefix uveden primo v zalohach, staci jej zmenit v kontejneru:
```
docker run --rm -it --entrypoint /bin/sh quay.io/canarytrace/installer:7.10.0


/etc/newman # ls /etc/postman/
Canarytrace_elastic7.x.postman_collection.json
kibana-7.x-dashboards.ndjson
kibana-7.x-index-patterns.ndjson
kibana-7.x-search.ndjson
kibana-7.x-visualizations.ndjson
```
a). po zmene spustit entrypoint
b). vytvorit si kontejner, provest zmeny a jako FROM pouzit `quay.io/canarytrace/installer:7.10.0`

**Reply by marekvesely-direct** [reply]
A je nejaky plan ze byste to vytahli do promenne? zase mi prijde skoda, kdyz to jde v canarytrace aplikaci zmenit pro Elastic pri vytvateni template, ale nejde to pro Kibanu.

**Reply by rdpanek** [reply]
Jako improvement to dava smysl to do jednoho z releasu zahrnout. Budeme releasovat nove dashboardy, tak by jsme mohli zahrnout i toto
